### PR TITLE
Miscellanous console output tweaking

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: utils/common/src/main/kotlin/ProcessCapture.kt [hunk 1]
```diff
@@ -129,9 +129,7 @@ class ProcessCapture(
         }
 
     init {
-        logger.info {
-            "Running '$commandLine' in '$usedWorkingDir'..."
-        }
+        logger.debug { "Running '$commandLine' in '$usedWorkingDir'..." }
```
Comment (mnonnenmacher) [inline utils/common/src/main/kotlin/ProcessCapture.kt:132]: I find this to be one of the most useful log lines when debugging failed runs, I wouldn't want it to appear only in debug.

File: scanner/src/main/kotlin/provenance/PackageProvenanceResolver.kt [hunk 2]
```diff
@@ -131,7 +131,7 @@ class DefaultPackageProvenanceResolver(
 
             else -> {
                 logger.info {
-                    "Could not find a stored artifact resolution result for package '${pkg.id.toCoordinates()}'," +
+                    "Could not find a stored artifact resolution result for package '${pkg.id.toCoordinates()}', " +
```
Comment (mnonnenmacher) [inline scanner/src/main/kotlin/provenance/PackageProvenanceResolver.kt:134]: Commit message nit: "after", not "before"

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: utils/common/src/main/kotlin/ProcessCapture.kt (hunk 1 above)
Comment (sschuberth) [reply utils/common/src/main/kotlin/ProcessCapture.kt]: Ok, I've dropped it.

**Review by mnonnenmacher — APPROVED** [review]

**Comment by sschuberth** [comment]
Merging despite the unrelated test failure.

**Pull request merged**